src/components/Link.tsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import type { HTMLProps, PropsWithChildren } from 'react';
+
+interface Props extends HTMLProps<HTMLAnchorElement> {
+    newTab: boolean;
+}
+
+export function Link({ newTab, children, ...rest }: PropsWithChildren<Props>) {
+    return (
+        <a rel="noopener noreferrer" target={newTab ? '_blank' : '_self'} {...rest}>
```

Comment:
I don't think the `rel="noopener noreferrer"` part is necessary if target is not `_blank`. But shouldn't hurt either.